style_editor.html

```diff
@@ -1178,6 +1178,26 @@
 var width;
 var height;
 
```

Comment:
How confident are we that all the input tags are for numerical values?

Reply:
Currently, they all are. There's no alpha input anywhere.